.github/workflows/tests_extra.yml

```diff
@@ -32,10 +32,10 @@ concurrency:
 
 jobs:
   latest_crds_contexts:
-    if: (github.repository == 'spacetelescope/jwst' && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'run scheduled tests')))
+    if: (github.repository == 'spacetelescope/jwst' && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'run extra tests')))
```

Comment:
Circling back to label name. Since it is already renamed, no reason to keep changing it back and forth. Human readable name is fine. ID would be too opaque even if unique.